src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -2117,31 +2179,46 @@ private static List<CompletionResult> GetResultForAttributeArgument(CompletionCo
             string argName = string.Empty;
             Ast argAst = completionContext.RelatedAsts.Find(static ast => ast is NamedAttributeArgumentAst);
             NamedAttributeArgumentAst namedArgAst = argAst as NamedAttributeArgumentAst;
-            if (argAst != null && namedArgAst != null)
+            AttributeAst attAst;
+            if (argAst is not null && namedArgAst is not null)
             {
-                attributeType = ((AttributeAst)namedArgAst.Parent).TypeName.GetReflectionAttributeType();
+                attAst = (AttributeAst)namedArgAst.Parent;
+                attributeType = attAst.TypeName.GetReflectionAttributeType();
                 argName = namedArgAst.ArgumentName;
                 replacementIndex = namedArgAst.Extent.StartOffset;
                 replacementLength = argName.Length;
             }
             else
             {
                 Ast astAtt = completionContext.RelatedAsts.Find(static ast => ast is AttributeAst);
-                AttributeAst attAst = astAtt as AttributeAst;
-                if (astAtt != null && attAst != null)
+                attAst = astAtt as AttributeAst;
+                if (astAtt is not null && attAst is not null)
```

Comment:
```suggestion
                if (astAtt is AttributeAst attAst)
```

Reply:
That's invalid syntax. attAst is already declared on line 2182.

Reply:
I see. I think we can remove duplicate check `astAtt is not null`.

Reply:
Yep, I've removed it.